FAIL
Kernel panic in run: PANIC at ../../threads/synch.c:234 in lock_release(): assertion `lock_held_by_current_thread (lock)' failed.
Call stack: 0xc01070bc 0xc010269c 0xc0109cd2 0xc0100e42
Translation of call stack:
0xc01070bc: debug_panic (.../../lib/kernel/debug.c:34)
0xc010269c: lock_release (...../../threads/synch.c:236)
0xc0109cd2: a_thread_func (...ity-donate-multiple2.c:72)
0xc0100e42: kernel_thread (..../../threads/thread.c:404)
